addons/website_hr_recruitment/views/website_pages_views.xml

```diff
@@ -12,13 +12,12 @@
             <attribute name="js_class">website_pages_list</attribute>
             <attribute name="type">object</attribute>
             <attribute name="action">open_website_url</attribute>
-
-            <!-- TODO could be a nice addition but editing "Department" crashes... -->
-            <!-- <attribute name="multi_edit">1</attribute> -->
+            <attribute name="multi_edit">1</attribute>
         </xpath>
 
         <field name="name" position="after">
             <field name="website_url"/>
+            <field name="company_id" invisible="1"/>
```

Comment:
Was not there already a company_id in the base view?

Reply:
@qsm-odoo sorry, I forgot to explain that on commit message :man_facepalming:
 
If user doesn't match the `groups` attribute, the field **with this attribute** will be removed from the view (not loaded/available for `domain`, `attrs`... computing) so if `company_id` is not added as "invisible", the edit of `department_id` (using it on its domain) will crash outside of multi company.

You can find a good explanation in [DLE's view refactoring](https://github.com/odoo/odoo/pull/95729#issuecomment-1257954879).

Reply:
Ok, good to know, I was not sure duplicating was the way to go :+1: